saleor/graphql/order/filters.py

```diff
@@ -27,7 +28,13 @@
 
 def filter_payment_status(qs, _, value):
     if value:
-        qs = qs.filter(payments__is_active=True, payments__charge_status__in=value)
+        if ChargeStatus.FULLY_REFUNDED in value:
+            qs = qs.filter(
+                Q(payments__is_active=True, payments__charge_status__in=value)
+                | Q(payments__charge_status=ChargeStatus.FULLY_REFUNDED)
+            )
+        else:
+            qs = qs.filter(payments__is_active=True, payments__charge_status__in=value)
```

Comment:
LGTM 👍 

Could be simplified a little:
```suggestion
        lookup = Q(payments__is_active=True, payments__charge_status__in=value)
        if ChargeStatus.FULLY_REFUNDED in value:
            lookup |=  Q(payments__charge_status=ChargeStatus.FULLY_REFUNDED)
        qs = qs.filter(lookup)
```